test/parallel.2.test_helper.bats

```diff
@@ -35,56 +37,60 @@ load 'test_helper/common'
   assert_failure
 }
 
-@test "run_until_success_or_timeout returns instantly on success" {
+@test "${TEST_NAME_PREFIX} run_until_success_or_timeout returns instantly on success" {
   SECONDS=0
   run_until_success_or_timeout 2 true
   [[ ${SECONDS} -le 1 ]]
   assert_success
 }
 
-@test "run_until_success_or_timeout waits for timeout on persistent failure" {
+@test "${TEST_NAME_PREFIX} run_until_success_or_timeout waits for timeout on persistent failure" {
   SECONDS=0
   ! run_until_success_or_timeout 2 false
   [[ ${SECONDS} -ge 2 ]]
   assert_failure
 }
 
-@test "repeat_in_container_until_success_or_timeout fails immediately for non-running container" {
+@test "${TEST_NAME_PREFIX} repeat_in_container_until_success_or_timeout fails immediately for non-running container" {
   SECONDS=0
   ! repeat_in_container_until_success_or_timeout 10 name-of-non-existing-container true
   [[ ${SECONDS} -le 1 ]]
 }
 
-@test "repeat_in_container_until_success_or_timeout run command in container" {
+@test "${TEST_NAME_PREFIX} repeat_in_container_until_success_or_timeout run command in container" {
   local CONTAINER_NAME
   CONTAINER_NAME=$(docker run --rm -d alpine sleep 100)
   SECONDS=0
   ! repeat_in_container_until_success_or_timeout 10 "${CONTAINER_NAME}" sh -c "echo '${CONTAINER_NAME}' > /tmp/marker"
   [[ ${SECONDS} -le 1 ]]
   run docker exec "${CONTAINER_NAME}" cat /tmp/marker
   assert_output "${CONTAINER_NAME}"
+
+  docker rm -f "${CONTAINER_NAME}"
```

Comment:
Same here, the container should already remove itself after completion of the `sleep`?

Are you just wanting to bring it down sooner for cleanup? There is no explicit name assigned, so it should not cause any problems, the container will just run for 100 seconds and be seen in `docker ps` until then.

Reply:
See https://github.com/docker-mailserver/docker-mailserver/pull/2822#discussion_r996285441 :)